### PR TITLE
Ulrich Windl's suggestions for Virtualization guide (bsc#1179924)

### PR DESCRIPTION
**Pull request by tbazant**
### Description

Beta-tester Ulrich  Windl suggested a few improvements for the Virtualization guide. This update includes all of them.

### Are there any relevant issues/feature requests?

* bsc#1179924

### To which product versions do the changes apply?

The first column is to be filled by the requester, the second column is to be filled by the doc team after the changes have been backported to the maintenance branches.

| Code 15     | Backport Done
| ----------  | ---------- 
| [x] 15 SP3  | (`master` only)   
| [x] 15 SP2  | [ ] 
| [x] 15 SP1  | [ ] 
| [x] 15 SP0  | [ ] 

| Code 12     | Backport Done
| ----------  | ---------- 
| [ ] 12 SP5  | [ ] 
| [ ] 12 SP4  | [ ] 
| [ ] 12 SP3  | [ ] 
| [ ] 12 SP2  | [ ]

### CODE REVIEW

File: xml/xen_virtualization_vhost.xml [hunk 1]
```diff
@@ -156,10 +156,23 @@
...
+      Edit <filename>/etc/default/grub</filename> and adjust the
+      <option>GRUB_CMDLINE_XEN</option> option so that it includes
+      <literal>dom0_mem=<replaceable>MEM_AMOUNT</replaceable></literal>.
+      Replace <replaceable>MEM_AMOUNT</replaceable> with the maximum amount of
+      memory to allocate to &dom0;. Add <command>K</command>,
+      <command>M</command>, or <command>G</command>, to specify the size unit.
+      For example:
+     </para>
+<screen>GRUB_CMDLINE_XEN="dom0_mem=2G"</screen>
+    </step>
```
Comment (jfehlig) [inline xml/xen_virtualization_vhost.xml:174]: You added the step describing use of /etc/default/grub, but didn't remove the yast steps. IIUC those should be removed right?

**Review by jfehlig — COMMENTED** [review]
I agree with Ulrich's suggestions. Only one small nit. See the inline comment for details.

File: xml/xen_virtualization_vhost.xml [hunk 2]
```diff
@@ -164,10 +156,23 @@
     <step>
      <para>
       At &dom0;, type <command>xl info</command> to view the amount of memory
-      that is available on the machine. &dom0;'s current memory allocation can
+      that is available on the machine. &dom0;'s cur rent memory allocation can
```
Comment (lproven) [inline xml/xen_virtualization_vhost.xml:159]:
```suggestion
      that is available on the machine. &dom0;'s current memory allocation can
```

Comment (lproven) [reply xml/xen_virtualization_vhost.xml:159]: This looks like an accidental change to me, and reverting it might be cleaner.

**Review by lproven — CHANGES_REQUESTED** [review]

File: xml/xen_virtualization_vhost.xml (hunk 1 above)
Comment (tbazant) [reply xml/xen_virtualization_vhost.xml:174]: a new commit fixes that, i forgot to push 1 change...

**Review by lproven — APPROVED** [review]

File: xml/xen_virtualization_vhost.xml (hunk 1 above)
Comment (jfehlig) [reply xml/xen_virtualization_vhost.xml:174]: Looks good now. Thanks!

**Pull request merged**